test/integration/Deploy.t.sol

```diff
@@ -25,7 +25,7 @@ contract TestDeployment is Test, DeploymentFixtures {
         vm.startPrank(csm.getRoleMember(csm.DEFAULT_ADMIN_ROLE(), 0));
         csm.grantRole(csm.RESUME_ROLE(), address(this));
         vm.stopPrank();
-        csm.resume();
+        if (csm.isPaused()) csm.resume();
```

Comment:
I would move the `grantRole` and the `prank` stuff in the if statement body to make it easier to read the trace.

Reply:
added if this is the only role to be granted